.github/workflows/ci_cd.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
       - name: Install style requirements
         run: |
-          pip install -r requirements_style.txt --disable-pip-version-check
+          pip install .[style] --disable-pip-version-check
```

Comment:
Here, we now also install the package itself since the style requirements are included in `pyproject.toml`. That's a slight negative to this approach.

Reply:
Not that I'm aware of, and didn't find anything from a quick search / look into pip options.